capsules/src/gpio.rs

```diff
@@ -51,17 +51,19 @@ use kernel::hil::gpio;
 use kernel::{AppId, Callback, Driver, Grant, ReturnCode};
```

Comment:
Need to update the comment above.

Reply:
I reverted the pins to the original numbering and updated the documentation in the component.